### PR TITLE
Merge popobs result

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: store/populations.go [hunk 1]
```diff
@@ -39,18 +40,44 @@ type PopObs struct {
...
+		}
 	}
-	return nil
+	resStr, err := (&jsonpb.Marshaler{}).MarshalToString(&res1)
+	if err != nil {
+		return err
+	}
+	out.Payload, err = util.ZipAndEncode(resStr)
+	return err
 }
```
Comment (pradh) [inline store/populations.go]: If the side-cache is empty do not unmarshall and remarshall?

File: store/populations.go [hunk 2]
```diff
@@ -39,18 +40,44 @@ type PopObs struct {
...
-		out.Payload, _ = util.ZipAndEncode("{}")
+		c, err := util.UnzipAndDecode(string(btRow[util.BtFamily][0].Value))
+		if err != nil {
+			log.Print(err)
+		}
+		jsonpb.UnmarshalString(string(c), &res2)
+
+		for k, v := range res2.Populations {
+			res1.Populations[k] = v
+		}
```
Comment (pradh) [inline store/populations.go]: To clarify, if there is an overlapping pop-id, here, we are picking the BT one?

I wonder if we should pick the side-cache?

File: store/populations.go (hunk 1 above)
Comment (shifucun) [reply store/populations.go]: refactored to only unzip and zip when both cache exist

File: store/populations.go (hunk 2 above)
Comment (shifucun) [reply store/populations.go]: Changed so side cache has higher priority

**Pull request merged**